src/familybot/lib/database.py

```
@@ -911,106 +946,197 @@ def cache_game_details_with_source(
+        for steam_id, value in FAMILY_USER_DICT.items():
+            friendly_name, discord_id = _parse_family_config_entry(value)
+            config_members_to_insert.append((steam_id, friendly_name, discord_id))
+
+        if config_members_to_insert:
+            cursor.executemany(
+                "INSERT OR IGNORE INTO family_members (steam_id, friendly_name, discord_id) VALUES (?, ?, ?)",
+                config_members_to_insert,
+            )
+            conn.commit()
+            logger.info(
+                f"Database: Migrated {len(config_members_to_insert)} family members from config.yml."
```

Ruff (0.15.7)

[warning] Logging statement uses f-string

(G004)